site/app/controllers/course/CourseMaterialsController.php

```diff
@@ -377,18 +446,43 @@ public function ajaxUploadCourseMaterialsFiles() {
 
                         foreach ($zfiles as $zfile) {
                             $path = FileUtils::joinPaths( $upload_path, $zfile );
-                            $json[$path] = [
-                                'checked' => '1',
-                                'release_datetime' => $release_time
-                            ];
+                            if(!(is_null($sections))){
+                                $sections_exploded = @explode(",", $sections);
```

Comment:
Please remove the `@` and properly set `$sections` to a countable above here if it is not when it hits this line.